packages/Backend/permissions/onUpdate.test.js

```diff
@@ -0,0 +1,53 @@
+import * as admin from 'firebase-admin';
+import { config } from 'firebase-functions';
```

Comment:
For now, let's either remove this file and create a separate ticket to add tests for it later, or keep working on it until we get the tests working. Linting errors and problems with this file though are causing this pull request to not get a pass from the CI.